rune/libenclave/init_linux.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/containerd/console"
 	enclaveConfigs "github.com/inclavare-containers/rune/libenclave/configs"
 	"github.com/inclavare-containers/rune/libenclave/intelsgx/preload"
-	"github.com/inclavare-containers/rune/libenclave/internal/runtime/pal"
+	enclave_runtime_pal "github.com/inclavare-containers/rune/libenclave/internal/runtime/pal"
 	"github.com/opencontainers/runc/libcontainer/cgroups"
```

Comment:
Good work and thank you for your contribution. I don't think it is necessary to add `enclave_runtime_pal`. Can you show the reason?

Reply:
@haosanzi Now we call the function `enclave_runtime_pal.Loadbinary(config.EnclaveConfig.Enclave.Path)` of package enclave_runtime_pal, but url of the package is `github.com/inclavare-containers/rune/libenclave/internal/runtime/pal`. It is not end by  enclave_runtime_pal. So it is not in the habit of golang. My IDE fixed it automatically andi think it is more readable than before.
https://github.com/alibaba/inclavare-containers/blob/master/rune/libenclave/init_linux.go#L101

Reply:
@lifeng68 Thank you for your reply. It is more readable than before. Could you make another commit to solve the readability issue and this commit is to solve the process name encoding error?

Reply:
ok, will update soon.

Reply:
@haosanzi fixed